autoPlay.user.js

```diff
@@ -251,7 +258,23 @@ function disableParticles() {
 	}
 }
 
+function updateCurrentTime() {
+    cHours = cTime.getUTCHours();
```

Comment:
Yeah, We need reintialize it every time updateCurrentTime is called.
I'll fix it.